### DISCUSSION TITLE
Doubt about capturing points

### DISCUSSION THREAD

**Discussion by fabriciosautner** [Agontuk/react-native-geolocation-service#344] [General]
Hello,

I didn't find it in the documentation and here at ISSUES I found scattered answers.

Can you explain better how the point capture routine works with distanceFilter and/or with interval? For example: who has priority over whom.

To help, my questions are:
-> If I have distanceFilter: 3 and interval:2000, then it doesn't matter if it arrives in 2 seconds, if the distance of the last captured point is less than 3 meters, no point will be captured, right?
-> If I set distanceFilter: 0 and interval: 2000, then every 2 seconds a GPS point will be taken.
-> At higher speeds could I get less GPS points? Ex: at 30 km/h get GPS points every 20 meters.

Thank you in advance for your attention.

**Comment by Agontuk** [comment]
1) `interval: 0`, `distanceFilter: X`
You will only get updates if user moves `X` meters

2) `interval: X`, `distanceFilter: 0`
You will only get updates after every `X` milliseconds

3) `interval: X`, `distanceFilter: X`
You will only get updates if user moves `X` meters & `X` milliseconds have already passed
